### DISCUSSION TITLE
How to remove app in the trial version?

### DISCUSSION THREAD

**Discussion by joyshmitz** [gravitational/teleport#6851] [Q&A]
this is possible at all, did not find a solution in the UX,
in the terminal stopped the process for the application, the icon continues to be present on the UX panel "Applications"
apparently there is a delay, after ~ an hour it cleared.

**Comment by Valien** [comment]
If you have access to the node that has the app access configured you can just delete that information from its `teleport.yaml` file and then restart the Teleport service. Once the node refreshes and reconnects back to the Auth server the app should be gone.

**Reply by joyshmitz** [reply]
Thank you
